packages/toolpad-app/src/components/DialogForm.tsx

```diff
@@ -1,7 +1,36 @@
+import React from 'react';
```

Comment:
The correct import path

```suggestion
import * as React from 'react';
```